test/e2e/handler/simple_bridge_and_bond_test.go

```diff
@@ -118,9 +118,9 @@ func bondUpWithEth1Eth2AndVlan(bondName string) nmstate.State {
 
 var _ = Describe("NodeNetworkState", func() {
 	Context("when desiredState is configured", func() {
-		Context("with a linux bridge up with no ports", func() {
+		Context("with a linux bridge up with some ports", func() {
```

Comment:
You wrote on the commit message -
```
linux-bridge CNI configures vlan_filtering flag for veth pairs, so there shouldn't be a need for it here.
```
Can you please explain what is vlan filtering on veth pair and what problem it solves?
I mean, what if the CNI didn't configure the vlan filtering on the veth, would it cause any problem?

Reply:
Now when I think about this sentence, I see is as not relevant really, because this change doesn't affect it in any way.
But since you're asking, the way I understand it is that there are northbound and southbound bridge ports, northbound are the ports configured by nmstate, southbound are the veth pairs created by CNI that connect virt-launcher pod with the bridge.
These veth pairs need to have the vlan filtering configured in compliance to what we configure by nmstate (or the vlan_filtering script respectively) to be able to communicate